front/lib/actions/server.ts

```diff
@@ -48,7 +48,8 @@ export async function runActionStreamed(
   auth: Authenticator,
   actionName: DustRegistryActionName,
   config: DustAppConfigType,
-  inputs: Array<unknown>
+  inputs: Array<unknown>,
+  tracingRecords: Record<string, string>
```

Comment:
In connectors we call it `loggerArgs` maybe keep the same?

Reply:
We already have a const called `loggerArgs` in this function and while discussing it with Henry and Flavien, the name that very quickly emerged was `tracing`, hence this name.

Reply:
:+1: